src/collection.jl

```diff
@@ -88,25 +145,25 @@ function fitscollection(basedir::String;
                 # filtering out comment columns
                 _keys = filter(k -> k ∉ exclude_key, keys(header_data))
                 _values = (header_data[k] for k in _keys)
-                push!(df, (path = path, name = name, hdu = index, zip(Symbol.(_keys), _values)...); cols = :union)
+                push!(collection, (path = path, name = name, hdu = index, zip(Symbol.(_keys), _values)...); cols = :union)
             end
             close(fits_data)
         end
     end
-    return df
+    return collection
 end
 
 
 """
-    arrays(df::DataFrame)
+    arrays(collection::DataFrame)
 
 Generator for arrays of images of entries in data frame.
```

Comment:
These docstrings are much too sparse. It should be clear to the user exactly *what* is being output by the iteration, *how* the iteration is occurring, and *what* the options are, if any. Be very clear that this function returns data loaded by FITSIO using `CCDReduction.get_data` into an `Array`. It utilizes the `path` and `hdu` from the collection. 

This will really make clear how these 3 functions are different, as well as how each one operates.